generators/server/index.js

```diff
@@ -438,7 +438,7 @@ module.exports = class JHipsterServerGenerator extends BaseBlueprintGenerator {
             'backend:info': './gradlew -v',
             'backend:doc:test': `./gradlew javadoc ${excludeWebapp}`,
             'backend:nohttp:test': `./gradlew checkstyleNohttp ${excludeWebapp}`,
-            'backend:start': './gradlew',
+            'backend:start': `./gradlew ${excludeWebapp}`,
```

Comment:
From my understanding '-x webapp' includes webapp instead of removing.
Not a gradle user.

Reply:
-x is correct